libmscore/instrument.h

```diff
@@ -123,6 +123,9 @@ class Channel {
 
       bool _userBankController = false;
 
+      mutable std::vector<MidiCoreEvent> _init;
+      mutable bool _mustUpdateInit = false;
```

Comment:
Wouldn't it make sense to set this flag to `true` for new `Channel` objects? The constructor doesn't seem to fill the init list with something meaningful so an update should be anyway required. As an alternative, the flag could be set to `true` in `Channel` constructor after initializing `_init` with empty events.

Reply:
Yes, that's true.